core/db_models/EEM_Base.model.php

```diff
@@ -5585,16 +5585,7 @@ public function item_name($quantity = 1)
      * happen?) they can add a hook onto 'filters_hook_espresso__{className}__{methodName}' (eg,
      * filters_hook_espresso__EE_Answer__my_great_function) and accepts 2 arguments: the object on which the function
      * was called, and an array of the original arguments passed to the function. Whatever their callback function
-     * returns will be returned by this function. Example: in functions.php (or in a plugin):
-     * add_filter('FHEE__EE_Answer__my_callback','my_callback',10,3); function
-     * my_callback($previousReturnValue,EE_TempBase $object,$argsArray){
-     * $returnString= "you called my_callback! and passed args:".implode(",",$argsArray);
-     *        return $previousReturnValue.$returnString;
-     * }
-     * require('EEM_Answer.model.php');
-     * $answer=EEM_Answer::instance();
-     * echo $answer->my_callback('monkeys',100);
-     * //will output "you called my_callback! and passed args:monkeys,100"
```

Comment:
plz restore this example as well.
if your reason was because of the `echo $` then you can simply get rid of that var by changing this:

```php
     * $answer=EEM_Answer::instance();
     * echo $answer->my_callback('monkeys',100);
```

 to this:

```php
     * echo EEM_Answer::instance()->my_callback('monkeys',100);
```

Reply:
Yes, Restored with a little change.